plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -6268,4 +6342,29 @@ private List<Long> getSnapshotsIdsByCreationOrder(String tableName)
                 .map(row -> (Long) row.getField(idField))
                 .collect(toList());
     }
+
+    private String getFieldFromLatestSnapshotSummary(String tableName, String summaryFieldName)
+    {
+        String findLatestFieldValue = format("SELECT json_extract_scalar(CAST(SUMMARY AS JSON), '$.%s') FROM \"%s$snapshots\" ORDER BY committed_at DESC LIMIT 1", summaryFieldName, tableName);
```

Comment:
nit: we can probably inline the statement in the next statement.

Reply:
Sure, inlined it